documentation/feature/medical-system.md

```diff
@@ -61,27 +68,38 @@ Besides the 4 elements introduced by basic medical, advanced introduces the foll
 
 #### 3.1.1 Stopping bleeding
-In order to stop the bleeding, all injuries on every bodypart requires treatment. This is done by either applying a tourniquet to legs or arms as a temporarly solution, or by using bandages to stop the bleeding as a more permament fix. 
+
+In order to stop the bleeding, all injuries on every bodypart requires treatment. This is done by either applying a tourniquet to legs or arms as a temporarly solution, or by using proper bandages to stop the bleeding as a more permament fix. Depending on settings, wounds can also open by itself even if bandaged. This will happen more often if wrong type of bandage is used. To completely stop bleeding wounds should be stitched after bandaging by trained medic.
 
 #### 3.1.2 Vitals
+
 While a unit is bleeding however, the blood volume decreases which will result in a change of vitals.  Depending on the factors such as current blood volume, the blood loss rate, medication used, the blood pressure will start to drop. To counter this drop, also based upon the previously mentioned factors and others, the heart rate will adjust accordingly to attempt to keep blood pressure at safe levels. This means that for any patient it is required to keep an eye on the vitals. This is done through the interaction system by selecting check pulse or blood pressure on either the arms or head.
 
+Heart rate under 45BPM is considered **low**, different medications listed below will have different effects depending on heart rate.
+It should be noted, that too high or too low heart rate can cause cardiac arrest. CPR should be done immediately if it happens.
+
+It should also be noted, that too high or too low blood pressure will often cause unconsciousness, and in some situations will cause cardiac arrest and even death.
+
 #### 3.1.3 Medication
+
 To stabilize the vitals and to counter for example pain, a player/medic can use medication. Advanced medical has 3 different medications available:
 
 * Atropine
 * Morphine
 * Epinephrine
 
-Atropine is a vagolytic and anticholinergic drug which in low dosages reduces heart rate but in high dosages increases it, countering effects of organophosphate poisoning (in NBC scenarios; anticholinesterase poisoning) and symptomatic bradycardia (in post-ROSC care and resuscitative medicine).
+Atropine is a vagolytic and anticholinergic drug which counters the effects of organophosphate poisoning (in NBC scenarios; anticholinesterase poisoning) and symptomatic bradycardia (in post-ROSC care and resuscitative medicine). In other words it can be used to counter effects of epinephrine as it will always lower the heart rate. 
```

Comment:
While in game it counters the effects of epi, Atropine is not a counter medication for it. The last sentence here should probably be reworded.